mmv1/templates/terraform/examples/chronicle_watchlist_basic.tf.tmpl

```diff
@@ -0,0 +1,21 @@
+provider "google-beta" {
+ user_project_override = true
+}
+
+resource "google_chronicle_watchlist" "{{$.PrimaryResourceId}}" {
+ provider = "google-beta"
+ location = "us"
+ instance = "{{index $.TestEnvVars "chronicle_id"}}"
+ watchlist_id = "{{index $.Vars "name"}}"
+ description = "{{index $.Vars "description"}}"
+ display_name = "{{index $.Vars "name"}}"
+ multiplying_factor = 1
+ entity_population_mechanism {
+  manual {
+
+  }
+ }
+ watchlist_user_preferences {
+  pinned = true
+ }
+}
```

Comment:
The indentation should be 2 white spaces.

Reply:
Done!